labs/lab3-paxos/src/dslabs/paxos/PaxosServer.java

```diff
@@ -39,7 +39,14 @@ public void init() {
        -----------------------------------------------------------------------*/
 
     /**
-     * Return the status of a given slot in the servers's local log.
+     * Return the status of a given slot in the server's local log. This
+     * includes acceptances on this node.
+     * 
+     * Order of precedence, in descending order, is:
```

Comment:
"Precedence" is a little vague? Maybe spelling out the logic in a paragraph would be better.